pages/index.js

```diff
@@ -19,7 +29,27 @@ export default function HomePage({ handleToggleOwned, plants }) {
           </li>
         ))}
       </ul>
+      )}
       <Link href="/myplants">My Plants</Link>
     </main>
   );
 }
+
+
+const StyledInfoText = styled.p`
+  color: var(--green-main);
+  background-color: var(--gray);
+  margin-top: 10px;
+  padding: 40px 40px;
```

Comment:
Since you have the same spacing for `top` and `left`, `right` and `bottom` will be the same. Therefore you can simply write: `padding: 40px;`